boards/atmel/sam/sam4e_xpro/sam4e_xpro.dts

```diff
@@ -53,24 +53,24 @@
 		#gpio-cells = <2>;
 		gpio-map-mask = <0xffffffff 0xffffffc0>;
 		gpio-map-pass-thru = <0 0x3f>;		/*           Shared */
-		gpio-map =	<0  0 &piob  2 0>,	/* AFE AD0          */
-				<1  0 &piob  3 0>,	/* AFE AD1          */
-				<2  0 &pioa 24 0>,	/* GPIO             */
-				<3  0 &pioa 25 0>,	/* GPIO             */
-				<4  0 &pioa 15 0>,	/* TIOA1            */
-				<5  0 &pioa 16 0>,	/* TIOB1            */
-				<6  0 &pioa 11 0>,	/* WKUP7            */
-				<7  0 &piod 25 0>,	/* GPIO             */
-				<8  0 &pioa  3 0>,	/* TWD0        EXTx */
-				<9  0 &pioa  4 0>,	/* TWCK0       EXTx */
-				<10 0 &pioa 21 0>,	/* RXD1             */
-				<11 0 &pioa 22 0>,	/* TXD1             */
-				<12 0 &piob 14 0>,	/* SPI(NPCS1)       */
-				<13 0 &pioa 13 0>,	/* SPI(MOSI)   EXTx */
-				<14 0 &pioa 12 0>,	/* SPI(MISO)   EXTx */
-				<15 0 &pioa 14 0>;	/* SPI(SCK)    EXTx */
-							/* GND              */
-							/* +3.3V            */
+		gpio-map = <0 0 &piob 2 0>,	/* AFE AD0          */
```

Comment:
The comments still not correct. Align the 3 boards and disable formatting for these files.

Reply:
@nandojve so at the moment linter will fail a PR if formatting does not comply once this PR (https://github.com/zephyrproject-rtos/zephyr/pull/92334) gets in.

The above mentioned PR will check files touched by that PR once the compliance check is executed.

At the moment you can disable linter with `// dts-format off` and `// dts-format on` between code block or just '// dts-format off' and all lines in the file under that comment will not be formatted or checked.

If this way of disabling the linter is not sufficient let me know why so we can consider other solutions

Reply:
@kylebonnici ,

Ok, so maybe I open a PR to fix this before yours be merged, right ?